src/ssl.c

```diff
@@ -13603,9 +13603,11 @@ static int Set_CTX_max_proto_version(WOLFSSL_CTX* ctx, int ver)
         }
         /* Update the method */
         switch (ver) {
+    #ifndef NO_OLD_TLS
```

Comment:
Can you move `#ifndef NO_TLS` at line 13610 to above at 13606? I think the `case SSL2_VERSION:` should be included in the NO_TLS logic.